src/config/dhiwayClient.ts

```
@@ -0,0 +1,112 @@
+
+// =============================================
+// Dhiway API Configuration
+// =============================================
+
+const DHIWAY_BASE_URL = import.meta.env.VITE_DHIWAY_API_URL;
+const DHIWAY_TOKEN = import.meta.env.VITE_DHIWAY_TOKEN;
+
+/**
+ * Extract error message from Dhiway API error response
+ */
+const extractErrorMessage = (error: any): string => {
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Replace `'Accept'` with `Accept`

(prettier/prettier)